### PR TITLE
Convert uncurried functions

### PR DESCRIPTION
**Pull request by JordanMartinez**
The snapshot currently errors because the lib doesn't include the corresponding Scheme FFI file.

### CODE REVIEW

**Comment by f-f** [comment]
Should we have CI pass before reviewing?

**Comment by JordanMartinez** [comment]
> Should we have CI pass before reviewing?

Yes, CI should pass before we review. I'm not sure what's causing this issue, but I'll continue working on it.

File: src/PureScript/Backend/Chez/Convert.purs [hunk 1]
```diff
@@ -142,10 +142,15 @@ codegenExpr codegenEnv@{ currentModule } (NeutralExpr s) = case s of
...
-  UncurriedApp _ _ ->
-    S.Identifier "uncurried-app"
-  UncurriedAbs _ _ ->
-    S.Identifier "uncurried-abs"
+  UncurriedApp fn args ->
+    S.List $ Array.cons (codegenExpr codegenEnv fn) $
+      (codegenExpr codegenEnv <$> args)
+  UncurriedAbs args body ->
+    S.List
+      [ S.Identifier "scm:lambda"
```
[the remark this answers is not in the document]
Comment (JordanMartinez) [reply src/PureScript/Backend/Chez/Convert.purs]: Thanks.

**Review by f-f — APPROVED** [review]

**Comment by JordanMartinez** [comment]
So... I was wondering why the error message I was getting in CI wasn't the same as running the actual scheme command. Turns out there was a copy-paste bug in `node-execa`: https://github.com/JordanMartinez/purescript-node-execa/pull/6

**Pull request merged**